src/autocomplete.js

```diff
@@ -197,7 +197,14 @@ class Autocomplete {
     }
 
     $updateGhostText(completion) {
-        var prefix = util.getCompletionPrefix(this.editor);
+        // Ghost text can include characters normally not part of the prefix (e.g. whitespace).
```

Comment:
do you know why `getCompletionPrefix` in utils doesn't consider whitespaces? Should we update that too?